src/components/Comments/WriteComment.js

```diff
@@ -137,7 +136,7 @@ const WriteComment = ({
           disabled={!isCommentEnabled || loadingUser || loadingPost}>
           <MemoSendComment
             fillBackground={
-              !isCommentEnabled || loadingUser || loadingPost ? '#C4C4C4' : colors.bondi_blue
+              !isCommentEnabled || loadingUser || loadingPost ? COLORS.gray1 : COLORS.holyTosca
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The fill color of the `MemoSendComment` component has been updated. Ensure that these new colors are consistent with your design system.
```diff
- !isCommentEnabled || loadingUser || loadingPost ? '#C4C4C4' : colors.bondi_blue
+ !isCommentEnabled || loadingUser || loadingPost ? COLORS.gray1 : COLORS.holyTosca
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->